package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "performance",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Performance plugin from the WordPress Performance Group, which is a collection of standalone performance modules.",
+  "author": "The WordPress Contributors",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "performance",
+    "wordpress"
+  ],
+  "homepage": "https://wordpress.org/plugins/performance-lab/",
+  "repository": "git+https://github.com/WordPress/performance.git",
+  "bugs": {
+    "url": "https://github.com/WordPress/performance/issues"
+  },
+  "engines": {
+    "node": ">=10.0.0",
+    "npm": ">=6.9.0 <7"
```

Comment:
Why does npm need to be <7?

Reply:
I asked in the editor chat room.

Reply:
seems like some bug in vs 7 that conflicts with the "Appium" dependency (which is related to their mobile app build). I think we can remove this here just.

Reply:
Great catch, I've removed this upper limit.

Reply:
Actually, you should be more strict about npm version because it uses a different format for the package lock file. I guess `>= 7` might be a good start since you aren’t limited by WordPress core that still builds with npm 6.

Aside: in Gutenberg we encountered several issues when trying to upgrade npm from v6 to v7/8 mostly because of the breaking changes for handling  peer dependencies.